### PR TITLE
Update to 2.1.2 - update dependencies, docs URL

### PR DESCRIPTION
**Pull request by philip-khor**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by philip-khor** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,26 +17,33 @@ build:
 
 requirements:
   build:
-    - python
+    - python >=2.7
```
Comment (sbrugman) [inline recipe/meta.yaml]: `pandas-profiling` v2.x.y supports Python 3 and dropped Python 2 support

(more details https://python3statement.org/)

Comment (philip-khor) [reply recipe/meta.yaml]: Fixed. @sbrugman would you like to be listed as a recipe maintainer?

Comment (sbrugman) [reply recipe/meta.yaml]: Yes, thank you.

**Comment by philip-khor** [comment]
@conda-forge-admin, please rerender

**Comment by philip-khor** [comment]
@conda-forge-admin, please rerender

**Comment by isuruf** [comment]
@conda-forge-admin, please rerender

**Comment by philip-khor** [comment]
@conda-forge/pandas-profiling this PR should be ready for merge

**Pull request merged**